tests/webtest_tests.py

```diff
@@ -46,46 +46,8 @@ def test_profile_disabled(self):
 
 class TestAnUnregisteredUser(OsfTestCase):
 
-    def test_can_register(self):
-        # Goes to home page
-        res = self.app.get('/').maybe_follow()
-        # Clicks sign in button
-        res = res.click('Create an Account or Sign-In').maybe_follow()
-        # Fills out registration form
-        form = res.forms['registerForm']
-        form['register-fullname'] = 'Nicholas Cage'
-        form['register-username'] = 'nickcage@example.com'
-        form['register-username2'] = 'nickcage@example.com'
-        form['register-password'] = 'example'
-        form['register-password2'] = 'example'
-        # Submits
-        res = form.submit().maybe_follow()
-        # There's a flash messageset
-        assert_in('Registration successful. Please check nickcage@example.com '
-            'to confirm your email address.', res)
-
-    def test_sees_error_if_email_is_already_registered(self):
-        # A user is already registered
-        user = UserFactory(username='foo@bar.com')
-        # Goes to home page
-        res = self.app.get('/').maybe_follow()
-        # Clicks sign in button
-        res = res.click('Create an Account or Sign-In').maybe_follow()
-        # Fills out registration form
-        form = res.forms['registerForm']
-        form['register-fullname'] = 'Foo Bar'
-        form['register-username'] = 'foo@bar.com'
-        form['register-username2'] = 'foo@bar.com'
-        form['register-password'] = 'example'
-        form['register-password2'] = 'example'
-        # submits
-        res = form.submit().maybe_follow()
-        # sees error message because email is already registered
-        assert_in('has already been registered.', res)
-
     def test_cant_see_profile(self):
-        """Can't see profile if not logged in.
-        """
+        """Can't see profile if not logged in."""
```

Comment:
Use line comments instead of docstrings for tests: http://cosdev.readthedocs.org/en/latest/process/testing.html#general-testing-guidelines